src/pages/importCsv.tsx

```diff
@@ -566,12 +571,30 @@ const ImportCsv = () => {
                                 <Box
                                   sx={{
                                     padding: "5px",
-                                    background:
-                                      getAbbreviatedMonth(
+                                    background: (() => {
+                                      const month = getAbbreviatedMonth(
                                         subTopic?.metaInformation?.startDate
-                                      ) === "Jan"
-                                        ? "#C1D6FF"
-                                        : "#FFD6D6",
+                                      );
+                                      switch (month) {
```

Comment:
@Aar-if instead of switch case here, can we below code
const monthColors = {
  "Jan": "#99CCFF",
  "Mar": "#D9B2FF",
  "Apr": "#FFABAB",
  "May": "#FFABAB",
  "Jun": "#FFABAB",
  "Jul": "#FFABAB",
  "Aug": "#FFABAB",
  "Sep": "#FFABAB",
  "Oct": "#FFD6D6",
  "Nov": "#FFD6D6",
  "Dec": "#FFD6D6"
};

const getColorForMonth = (month) => monthColors[month] || "#FFD6D6";

// Example usage:
const color = getColorForMonth("Jan"); // "#99CCFF"